content/momentum/3/3-api/hooks-core-ec-ssl-ssl-ctx-fixup.md

```diff
@@ -13,23 +13,16 @@ ec_ssl_SSL_CTX_fixup — This hook provides an opportunity for modules to alter
 
 `#include "hooks/core/ec_ssl_SSL_CTX_fixup.h"`
 
-| `void **core_ec_ssl_SSL_CTX_fixup** (` | <var class="pdparam">closure</var>, |   |
-|   | <var class="pdparam">sslctx</var>, |   |
-|   | <var class="pdparam">type</var>, |   |
-|   | <var class="pdparam">domain</var>, |   |
-|   | <var class="pdparam">certfile</var>, |   |
-|   | <var class="pdparam">keyfile</var>, |   |
-|   | <var class="pdparam">clientca</var>, |   |
-|   | <var class="pdparam">cipher_list</var>`)`; |   |
-
-`void * <var class="pdparam">closure</var>`;
-`struct ssl_ctx_st * <var class="pdparam">sslctx</var>`;
-`int <var class="pdparam">type</var>`;
-`const char * <var class="pdparam">domain</var>`;
-`const char * <var class="pdparam">certfile</var>`;
-`const char * <var class="pdparam">keyfile</var>`;
-`const char * <var class="pdparam">clientca</var>`;
-`const char * <var class="pdparam">cipher_list</var>`;<a name="idp42798880"></a> 
+```
+void core_ec_ssl_SSL_CTX_fixup (void *closure,
+                                struct ssl_ctx_st *ctx,
+                                int type,
+                                const char *domain,
+                                const char *certfile,
+                                const char *keyfile,
+                                const char *clientca,
+                                const char *cipher_list);
+```
```

Comment:
Not sure what they were trying to do with the table format.  Even with adding a blank header line to fix it, it just doesn't look good.  I could see maybe having a 3rd column with the below descriptions.

There's a boatload of these broken tables that really need to be fixed.  What's there really reflects poorly on the state of our docs.